src/main/java/edit_user_public_profile_usecase/editUserPublicProfileDSGateway.java

```diff
@@ -0,0 +1,8 @@
+package edit_user_public_profile_usecase;
```

Comment:
I think findUser does the same as the getUser method in my gateway; let's discuss on Monday what we're going to name any shared dsGateway methods and assign who's going to do the concrete implemention in the data access class.

Reply:
I think it would be better to just use a general userDsGateway for better clean architecture, as Sohee and I are gonna do a similar thing for our group creation and edit group profile use cases!